docs/source/public/overview.rst

```diff
@@ -541,11 +541,11 @@ If the `fix network script <fix_networks>`_ was run, two alchemical networks nee
 
 If confidential ligand names were used, those can be replaced with generic ligand names. To do that you will need to add the flag ``--hide-ligand-names``:
 
-.. code-block:: bash
+thering.py --input_alchemical_network network_setup/alchemicalNetwork/alchemical_network.json --results-folder results_0 --results-folder results_1 --results-folder results_2 --hide-ligand-names
```

Comment:
Looks like an accidental formatting error got introduced here.

Reply:
Oops, yep fixed it!